static/js/upload_widget.js

```diff
@@ -61,8 +66,10 @@ var upload_widget = (function () {
                 input_error.hide();
             } else if (e.target.files.length === 1) {
                 var file = e.target.files[0];
-                if (file.size > 5 * 1024 * 1024) {
-                    input_error.text(i18n.t('File size must be < 5Mb.'));
+                if (file.size > max_file_upload_size * 1024 * 1024) {
+                     input_error.text(i18n.t('File size must be < __max_file_size__Mb.', {
+                        max_file_size: max_file_upload_size,
```

Comment:
Not sure if there's supposed to be a comma at the end of this line. Correct me if I'm wrong.